resources/public/api/conf/1.0/schemas/retrieve_coding_out_underpayments_response.json

```diff
@@ -12,133 +12,331 @@
         "user"
       ]
     },
-    "selfAssessmentUnderPayments": {
-      "type": "array",
-      "description": "Array containing details of coding out Self Assessment underpayments.",
-      "minItems": 1,
-      "items": {
-        "type": "object",
-        "properties": {
-          "amount": {
-            "description": "The total coding out amount attributed to Self Assessment underpayments. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
-            "type": "number",
-            "multipleOf": 0.01,
-            "minimum": 0.00,
-            "maximum": 99999999999.99,
-            "example": "5000.99"
-          },
-          "relatedTaxYear": {
-            "description": "The tax year related to the underpayment.",
-            "type": "string",
-            "example": "2021-22"
-          },
-          "submittedOn": {
-            "description": "The date the coding out information was added. It is either the latest change date held within HMRC or the latest user submitted date. It is in the format YYYY-MM-DDThh:mm:ssZ.",
-            "type": "string",
-            "example": "2020-07-06T09:37:17Z"
+    "taxCodeComponents": {
+      "type": "object",
+      "description": "Object containing the tax code components",
+      "properties": {
+        "selfAssessmentUnderPayment": {
+          "type": "array",
+          "description": "Array containing details of coding out Self Assessment underpayments.",
+          "minItems": 1,
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "description": "The ID for the Self Assessment underpayment.",
+                "type": "number",
+                "minimum": 1.00,
+                "maximum": 99999999999.99
```

Comment:
999999999999999 not 99999999999.99
same for all ID's (no decimals) in min or max